### DISCUSSION TITLE
Inactive vs Delete

### DISCUSSION THREAD

**Discussion by nsetyo** [anonaddy/anonaddy#266] [Q&A] [answered]
What's the difference between inactive and deleted? As far as I know, we can also restore the soft deleted

**Comment by willbrowningme** [comment, marked as answer]
When an alias is deactivated (inactive) then it will silently `DISCARD` any messages that are sent to it, the sender of the message **will not be notified**.

When an alias is deleted then it will `REJECT` any messages that are sent to it and the sender will receive a rejection message `5.7.1 Recipient address rejected: Address does not exist`.

Yes you can restore an alias that has been deleted.

I will try to make all this more obvious and intuitive when I redesign the user interface.
